src/uct/ib/ud/base/ud_iface.c

```diff
@@ -305,6 +305,14 @@ uct_ud_iface_create_qp(uct_ud_iface_t *self, uct_ud_iface_config_t *config)
     return UCS_ERR_INVALID_PARAM;
 }
 
+void uct_ud_iface_complete_init(uct_worker_h worker, uct_ud_iface_t *iface, 
```

Comment:
can we move the timer creation here as well so ud constructores would not have to take a lock?